components/headerNav/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ComponentWrapper } from '../../styles/containers'
+import { colors } from '../../styles/colors'
+import { breakpoints } from "../../styles/breakpoints";
+import Link from 'next/link'
+import styled from 'styled-components'
+
+const ListContainer = styled.ul`
+  display: flex;
+  height: 100%;
+  flex-direction: row;
+  justify-content: flex-end;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`;
+const ListItem = styled.li`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  &:hover {
+    background-color: ${colors.fadedPrimaryBlue};
+    cursor: pointer
+  }
+`;
+const Anchor = styled.a`
+  display: flex;
+  color: ${colors.primaryBlue};
+  text-align: center;
+  padding: 0 20px;
+  ${breakpoints("padding", "", [
+  { 1200: "0 16px" },
+  { 800: "0 14px" },
+  { 600: "0 12px" },
+  { 450: "0 10px" }
+])};
+  font-size: 1rem;
+  text-decoration: none;
+`;
+const Logo = styled.img`
+  height: 2.8125vw;
+  margin-top: 0.46875vw;
+  ${breakpoints("height", "px", [
+  { 1200: 35 },
+  { 800: 30 },
+])};
+  ${breakpoints("margin-top", "px", [
+  { 1200: 5 },
+  { 800: 7.5 },
+])};
+  object-fit: contain;
+
+  &:hover {
+    cursor: pointer
+  }
+  `;
+
+const HeaderNav = () => {
+  return (
+    <ComponentWrapper height="3.9375vw" minHeight="45px" maxHeight="7vh" width="94vw" margins="auto" direction="row" justify="space-between" >
```

Comment:
Wouldn't a cleaner method be to wrap the Component Wrapper as a new styled component and include these stylings as a styled component.

```
const HeaderWrapper = styled(ComponentWrapper)`
  height=3.9375vw;
  minHeight=45px;
  maxHeight=7vh;
  width=94vw;
  margins=auto;
  direction=row;
  justify=space-between;
`
```

Reply:
@FB-Reda It's rly a matter a code design opinion. We can set a standard now if you want moving forward. Right now we have 2 global containers and they have default props but are set up to accept props inline, which is convention for mobile dev which is probs why I autopiloted towards that direction. We can also do what you're suggesting and use inheritance to create specific ones in each component file. I don't really have an affinity for one over the other so if you have a strong opinion on the matter then we can do that